src/olympia/translations/tests/test_commands.py

```diff
@@ -71,12 +71,12 @@ def test_basic(self, index_addons_mock):
             == 'foo http://bar.com'
         )
 
-        # That one should not have changed.
+        # We didn't double-escape that one.
         self.translation_with_already_escaped_html.reload()
         assert (
             self.translation_with_already_escaped_html.localized_string
             == self.translation_with_already_escaped_html.localized_string_clean
-            == 'foo &lt;a href=&quot;http://bar.com&quot;&gt;http://bar.com&lt;/a&gt;'
+            == 'foo &lt;a href="http://bar.com"&gt;http://bar.com&lt;/a&gt;'
```

Comment:
Minor behavior change: we no longer escape quotes that don't need to be escaped because they are not inside quotes themselves. This is something to consider in general, our escaping here is not context-aware, and shouldn't be relied upon to display HTML attributes...

Reply:
`nh3` has `clean_text()` for when we truly don't want HTML, but it would double-escape in this scenario. I'll check whether we have some places in addons-server that could benefit from that.